portfolio/src/main/webapp/script.js

```diff
@@ -37,15 +37,21 @@ async function fetchComments() {
   const commentsObject = await response.json();
   const commentsContainer = document.getElementById('comments-list');
   commentsContainer.innerHTML = '';
-  commentsObject.comments.forEach(comment => {
-    commentsContainer.appendChild(createListElement(comment));
+  commentsObject.forEach(comment => {
+    commentsContainer.appendChild(createListElement(
+      comment.text, comment.author));
   });
 }
 
 /** Creates an <li> element containing text. */
```

Comment:
Can this comment be more specific? I think it's also a little outdated, since you are now also adding the author as a param.

Reply:
Done!